### DISCUSSION TITLE
Create private Trivy docker image

### DISCUSSION THREAD

**Discussion by igor-teresco** [aquasecurity/trivy#7950] [Q&A]
### Question

With the help of recurring update task in Jenkins we have an updated install of trivy.
How to create an own docker image using local updated files to use in other tasks locally?
Which local files and where to copy in the image, please?

### Target

None

### Scanner

None

### Output Format

None

### Mode

Standalone

**Comment by itaysk** [comment]
If I understand correctly, you are looking for this guide: https://trivy.dev/v0.57/docs/advanced/air-gap/#manual-cache-population

**Reply by igor-teresco** [reply]
I was wondering which files to copy in docker image to get it updated
